proto/environment/environment.proto

```diff
@@ -18,6 +18,10 @@ package bucketeer.environment;
 option go_package = "github.com/bucketeer-io/bucketeer/proto/environment";
 
 message EnvironmentV2 {
+  message Settings {
+    bool require_comment = 1;
+  }
```

Comment:
Why is this nested?

Reply:
To maintain consistency with other proto messages.
Also, this settings message belongs to Environment only. I don't see a specific reason not to be in the Environment message.

Reference.
https://github.com/bucketeer-io/bucketeer/blob/main/proto/account/account.proto#L42-L69

Reply:
I wanted to ask why `Settings` is necessary. We can simply add `require_comment` to `EnvironmentV2`.
If `Settings` is introduced, `name` and `description` can be also settings.

Reply:
We might need to add more settings in the new UI or if some users ask us. So it will be easier to manage it when we add more settings.

> If Settings is introduced, name and description can be also settings.

Name and description are general information, not exactly a setting IMHO. It's used for reference only different from the settings that will be used in validations, etc.